### DISCUSSION TITLE
How to get collaboration on GitHub pages site from demo template?

### DISCUSSION THREAD

**Discussion by jmshea** [jupyterlite/jupyterlite#447] [Q&A]
I'm pretty new here --- I only started using JupyterLite yesterday, so I apologize if this is answered somewhere, but I have been searching and experimenting without luck. 

I can't seem to figure out how to get real-time collaboration working on a GitHub site that I generated from the demo template. 

What I tried:
1) Adding "collaborative":true to either/both of /jupyter-lite.json and /lab/jupyter-lite.json

2) Adding "?room=xxx" where xxx is various strings or numbers

If anyone knows how to get this working, please let me know!

**Comment by jtpio** [comment]
Thanks @jmshea.

Here is an example of a JupyterLite website generated from the template with RTC enabled: https://github.com/jtpio/lit

For now RTC sessions are per document, so users must open a notebook of file with the same name. For example:

![image](https://user-images.githubusercontent.com/591645/149417738-82c2458d-ef48-43d3-9b2c-e1c30aae395f.png)

The content in JupyterLite is decentralized with each user keeping a copy of the document. This is quite different compared to stock JupyterLab, and there are still a couple of glitches to fix, which is why it is not enabled by default in the demo template.
